assets/src/edit-story/elements/shape/layer.js

```diff
@@ -13,87 +13,32 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
 /**
  * External dependencies
  */
-import styled from 'styled-components';
-import { useMemo } from 'react';
-
+import { __ } from '@web-stories-wp/i18n';
 /**
  * Internal dependencies
  */
 import { getMaskByType } from '../../masks';
-import { elementWithBackgroundColor } from '../shared';
-import StoryPropTypes from '../../types';
-import getBrightnessFromPattern from '../../utils/getBrightnessFromPattern';
-
-const TOO_BRIGHT = 0.5;
-
-const Container = styled.div`
-  display: flex;
-  flex-direction: row;
-  justify-content: center;
-  align-items: center;
-`;
-
-const ShapePreview = styled.div`
-  ${elementWithBackgroundColor}
-  width: ${({ width = 20 }) => width}px;
-  height: ${({ height = 20 }) => height}px;
-  margin-right: 8px;
-`;
-
-const ShapePreviewContainer = styled.div`
-  ${({ isTooBright }) =>
-    isTooBright &&
-    `
-      /* Using filter rather than box-shadow to correctly follow
-      * outlines of shapes.
-      */
-      filter: drop-shadow( 0 0 5px rgba(0, 0, 0, 0.5) );
-    `}
-`;
```

Comment:
The designs didn't have these drop-shadows so I removed them from all of the logos. Do we still need these?